packages/cli/lib/commands/info.js

```diff
@@ -0,0 +1,13 @@
+const envinfo = require('envinfo') // eslint-disable-line
+
+module.exports = async () => {
+  const data = await envinfo
+    .run({
+      System: ['OS', 'CPU'],
+      Binaries: ['Node', 'Yarn', 'npm'],
+      Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+      npmPackages: '?(@)gridsome{-*,/*}',
```

Comment:
```suggestion
      npmPackages: '?(@)gridsome{-*,/*,}',
```
This will also include the `gridsome` package.